Tests/test_file_jpeg2k.py

```diff
@@ -371,3 +372,46 @@ def test_crashes(test_file):
                 im.load()
             except OSError:
                 pass
+
+
+def test_custom_comment():
+    output_stream = BytesIO()
+    unique_comment = "This is a unique comment, which should be found below"
+    test_card.save(output_stream, "JPEG2000", comment=unique_comment)
+    output_stream.seek(0)
+    data = output_stream.read()
+    # Lazy method to determine if the comment is in the image generated
+    assert bytes(unique_comment, "utf-8") in data
```

Comment:
Ideally, we could test this by reading the comment from the image as well. So I've created PR #6909

Reply:
#6909 has been merged now, so I've created https://github.com/joshware/Pillow/pull/2